fetch.bs

```diff
@@ -1446,10 +1445,9 @@ is a <a>filtered response</a> whose
 
 <h3 id=authentication-entries>Authentication entries</h3>
 
-<p>An <dfn export>authentication entry</dfn> and a
-<dfn export>proxy-authentication entry</dfn> are tuples of username, password, and realm,
-associated with one or more <a for=/>requests</a>.
-
+<p>An <dfn export>authentication entry</dfn> and a <dfn export>proxy-authentication entry</dfn> are
+tuples of username, password, and realm, used for HTTP authentication, and associated with one or
+more <a for=/>requests</a>
```

Comment:
Trailing dot went missing. Also, I'd make it "HTTP and HTTP proxy authentication" maybe?